flowbased-computation/flowbased-computation-api/pom.xml

```diff
@@ -76,6 +81,11 @@
             <artifactId>powsybl-iidm-xml-converter</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>farao-crac-impl</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
dependency order

Reply:
Is it for tests ? Why would this module rely on crac-impl ?

Reply:
It is supposed to be scope runtime